### PR TITLE
perf: update SVGs to support new token name spec #211

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #211 

## Summary:

This commit reverts the revert for using the new Auro design token name spec. The revert happened due to a side-effect with auro-icon and the customSize feature. The auro-icon element was updated to allow for more flexibility in token use.

See https://github.com/AlaskaAirlines/auro-icon/pull/95

This update has been verified that regardless of version of tokens a consumer has installed, or none at all, the customSize feature will work as expected.

Changes to be committed:
modified:   src/data/icons.json

## Type of change:

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by jason-capsule42 — DISMISSED** [review]

File: src/_icons.scss [hunk 1]
```diff
@@ -41,7 +41,7 @@ $iconMap: (
 // Square large
 .ico_squareLarge {
   fill: currentColor;
-  height: $auro-size-lg;
+  height: var(--auro-size-lg, var(--ds-size-300, 1.5rem))
```
Comment (jason-capsule42) [inline src/_icons.scss:44]: I think these also should all be --ds not --auro. Looks like just this one got missed.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/_icons.scss (hunk 1 above)
Comment (blackfalcon) [reply src/_icons.scss:44]: No, that's intentional. That's part of the seamless fallback strategy. There is code out there that only knows to look for the `--auro` token. So putting `--auro` first keeps things working as expected, but allows people to switch over to `--ds` and things will still work.

Comment (blackfalcon) [reply src/_icons.scss:44]: Also, there is no `--ds-size-lg`, that was removed with the new release.

Comment (jason-capsule42) [reply src/_icons.scss:44]: Got it, I see so the fallback for --auro is first the -ds-size-300 then the SASS var.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**